tasks/sbd_watchdog.yml

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Configure SBD watchdog
+  block:
+    - name: SBD - Configure and unload watchdog kernel modules from blocklist
+      block:
+        - name: "SBD - Configure watchdog kernel module blocklist"
+          ansible.builtin.lineinfile:
+            path: "/etc/modprobe.d/{{ item }}.conf"
+            create: yes
+            mode: 0644
+            regexp: "^(options|blacklist) {{ item }}"
+            line: "blacklist {{ item }}"
+            state: present
+          loop: "{{ ha_cluster.sbd_watchdog_modules_blocklist | d([]) }}"
+          register: __ha_cluster_sbd_watchdog_modules_blocklist_config
+
+        - name: "SBD - Unload watchdog kernel modules from blocklist"
+          community.general.modprobe:
+            name: "{{ item }}"
+            state: absent
+          loop: "{{ ha_cluster.sbd_watchdog_modules_blocklist | d([]) }}"
+          register: __ha_cluster_sbd_watchdog_blocklist_unload_output
+
+    - name: SBD - Configure and load watchdog kernel module
+      block:
+        - name: "SBD - Configure watchdog kernel modules"
+          ansible.builtin.lineinfile:
+            path: "/etc/modules-load.d/{{ item }}.conf"
+            create: yes
+            mode: 0644
+            regexp: "^{{ item }}"
+            line: "{{ item }}"
+            state: present
+          loop: "{{ ha_cluster.sbd_watchdog_modules | d([]) }}"
+          register: __ha_cluster_sbd_watchdog_modules_load_config
+
+        - name: "SBD - Load watchdog kernel modules"
+          community.general.modprobe:
+            name: "{{ item }}"
+            state: present
+          loop: "{{ ha_cluster.sbd_watchdog_modules | d([]) }}"
+          register: __ha_cluster_sbd_watchdog_modules_load_output
+
+  when:
```

Comment:
please put the `when` statement between the `name` and the `block` for readability (I know there are other places in the role where the `when` comes at the end, but ansible-lint 6.x is cracking down on this in the interest of readability)

Reply:
I went with "it's done this way everywhere" but also was tempted to format it like you say ;)
done